examples/SinusoidInterface/sinusoid_example.jl

```diff
@@ -0,0 +1,85 @@
+# # Fitting parameters of a sinusoid
```

Comment:
Maybe I'm missing something, but does this file need to be here? It doesn't seem to serve any purpose.

Reply:
I've already deleted this! forgot to push the change - sorry about that!